samples/ehr-flask/katanemo/integrations/flask_auth.py

```diff
@@ -0,0 +1,69 @@
+import json
+import katanemo_sdk
+import katanemo_auth_sdk
+import requests
+from flask import request as flask_request
+
+class KatanemoFlaskAuth:
+    
+    def __init__(self, app):
+        self.app = app
+        self.api_client = None
+        self.auth_client = None
+        self.api_access_token = None
+        self.client_id = None
+
+    def register(self, client_name, service_id, client_id, client_secret):
+        self.client_id = client_id
+        self.client_secret = client_secret
+        access_api_client = katanemo_sdk.AccessControlApi(self._get_access_api_client())
+        req = katanemo_sdk.OAuthTokenRequest(clientId=client_id, clientSecret=client_secret)
+        resp = access_api_client.get_short_term_token(req)
+        self.api_access_token = resp.access_token
+
+    def authorize_request(self, access_token, request_path, http_method, request_body):
+        auth_api_client = self._get_auth_client()
+
+        req = katanemo_auth_sdk.AuthorizationRequest(
+            token=access_token,
+            path=request_path,
+            http_method=http_method,
+            request_body=request_body,
+        )
+        auth_api_client.authorize_request(req)
+    
+    def authorize(self, redirect_uri, service_id):
```

Comment:
We need to declare our client API calls better. For example the OAuth2.0 /authorize call should be annotated as such. Else developers will be confused on what to do when...